src/blocks/extensions/quick-action-sidebar/components/blocks.js

```diff
@@ -8,12 +8,15 @@ import DraggableBlock from './draggable-block';
 
 const Blocks = () => {
     const blocks = getBlockTypes();
-	const { blockInsertionPoint, getBlockRootClientId } = useSelect( ( select ) => {
+	const { blockInsertionPoint, getSelectedBlockClientId, getSelectedBlockAllowedBlocks, getBlockRootClientId } = useSelect( ( select ) => {
 		const { index } = select( 'core/block-editor' ).getBlockInsertionPoint();
```

Comment:
```suggestion
        const blockEditor = select( 'core/block-editor' );
		const { index } = blockEditor.getBlockInsertionPoint();
```
blockEditor variable, we can use in line number 13,14,15 also

Reply:
updated